k9mail/src/main/java/com/fsck/k9/K9.java

```diff
@@ -237,6 +237,7 @@
     private static boolean mWrapFolderNames = false;
     private static boolean mHideUserAgent = false;
     private static boolean mHideTimeZone = false;
+    private static boolean mHideHostnameWhenConnecting = false;
```

Comment:
We're already mixing field names with and without prefixes. Please don't introduce more with the `m` prefix.

Reply:
I'll submit a separate PR to remove all the `m`s and rebase this onto it.